code/modules/organs/wound.dm

```diff
@@ -243,6 +264,9 @@
 	return null //no wound
 
 /** CUTS **/
+/datum/wound/cut/bleeding()
+	return ..() && wound_damage() >= 5
```

Comment:
If you're going to do it this way with an override and ..() call this should be `||`, not `&&`.

See https://github.com/Baystation12/Baystation12/pull/12755